src/platform/qt/AudioProcessorSDL.cpp

```diff
@@ -16,18 +16,19 @@ AudioProcessorSDL::AudioProcessorSDL(QObject* parent)
 {
 }
 
-AudioProcessorSDL::~AudioProcessorSDL() {
-	mSDLDeinitAudio(&m_audio);
-}
-
-void AudioProcessorSDL::setInput(mCoreThread* input) {
+void AudioProcessorSDL::setInput(std::shared_ptr<CoreController> input) {
 	AudioProcessor::setInput(input);
-	if (m_audio.core && input->core != m_audio.core) {
+	if (m_audio.core && input->thread()->core != m_audio.core) {
 		mSDLDeinitAudio(&m_audio);
-		mSDLInitAudio(&m_audio, input);
+		mSDLInitAudio(&m_audio, input->thread());
```

Comment:
`input` here is a variable shadowing the `input()` method, which returns the `->thread()` everywhere else. I should probably rename it so it doesn't shadow anymore.